apps/core/service.py

```diff
@@ -0,0 +1,70 @@
+"""Basic CRUD for services."""
+from typing import Type, TypeVar
+
+from fastapi.encoders import jsonable_encoder
+from pydantic import BaseModel
+from sqlalchemy import delete, select, update
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.engine import ChunkedIteratorResult, CursorResult
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from core.db import Base
+
+ModelType = TypeVar("ModelType", bound=Base)
+CreateSchemaType = TypeVar("CreateSchemaType", bound=BaseModel)
+UpdateSchemaType = TypeVar("UpdateSchemaType", bound=BaseModel)
+
+
+class AsyncCRUDBase:
```

Comment:
I cannot find `AsyncCRUDBase` class in old code and it seems to be unused. Isn't it?